smart_contracts.rst

```diff
@@ -442,7 +442,7 @@ Deposit more tokens into a channel. This will only increase the deposit of one o
 
 **Withdraw tokens from a channel**
 
-Allows a channel participant to withdraw tokens from a channel without closing it. Can be called by anyone. Can only be called once per each signed withdraw message.
+Allows a channel participant to withdraw tokens from a channel without closing it. Can be called by anyone. Can only be called once per each signed withdraw information.
```

Comment:
Hmm information also does not sound good here. Perhaps transaction? Since we are talking about onchain transactions anyway.

Reply:
"Signed transactions" sound too close to that transaction `msg.sender` signs.  I just go for `data` again.

Reply:
Replaced with "withdraw proof."

Reply:
Still not a fan of withdraw proof but can live with it :)